public/request/forum-topic/delete.php

```diff
@@ -13,8 +14,18 @@
     'topic' => 'required|integer|exists:ForumTopic,ID',
 ]);
 
+$forumTopicId = $input['topic'];
+
 /** @var ForumTopic $topic */
-$topic = ForumTopic::find($input['topic']);
-$topic->delete();
+$foundTopic = ForumTopic::find($forumTopicId);
+if ($foundTopic) {
+    $foundTopic->delete();
+
+    $foundAssociatedGame = Game::firstWhere('ForumTopicID', $forumTopicId);
+    if ($foundAssociatedGame) {
+        $foundAssociatedGame->ForumTopicID = null;
+        $foundAssociatedGame->save();
+    }
```

Comment:
shouldn't this happen automatically if we create a foreign key that has `->onDelete('set null')`?

Reply:
It should. I wasn't sure if we wanted to add a foreign key here, per the PR description:

> Normally, I'd reach for a foreign key with onDelete("set null"), but that might not be the correct approach given this comment on the Game model:

However, it seems like if we're going to store this data, that's probably the best approach. There's no timeline for when we'll have a forumable graph.